config.json

```diff
@@ -1,207 +1,207 @@
-      "uuid": "9460b65d-dc80-4a95-8782-b395d2cc979e",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 1,
-      "topics": null
+      slug: "word-count",
+      uuid: "9460b65d-dc80-4a95-8782-b395d2cc979e",
+      core: false,
+      unlocked-by: null,
+      difficulty: 1,
+      topics: null
     },
     {
-      "slug": "two-fer",
-      "uuid": "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 1,
-      "topics": null
+      slug: "two-fer",
+      uuid: "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e",
+      core: false,
+      unlocked-by: null,
+      difficulty: 1,
+      topics: null
     },
     {
-      "slug": "sieve",
-      "uuid": "732e4c98-675d-4b67-b672-37ef9c848733",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 2,
-      "topics": [
+      slug: "sieve",
+      uuid: "732e4c98-675d-4b67-b672-37ef9c848733",
+      core: false,
+      unlocked-by: null,
+      difficulty: 2,
+      topics: [
         "math",
         "vector"
       ]
     },
     {
-      "slug": "change",
-      "uuid": "6f2d8731-2c2c-4faa-a9bf-11f785c2e5bc",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 5,
-      "topics": [
-        "dynamic_programming",
+      slug: "change",
+      uuid: "6f2d8731-2c2c-4faa-a9bf-11f785c2e5bc",
+      core: false,
+      unlocked-by: null,
+      difficulty: 5,
+      topics: [
+        "dynamic-programming",
         "graph"
       ]
     },
     {
-      "slug": "pascals-triangle",
-      "uuid": "d5caff8b-ffaa-4055-9bb9-7d31a4480956",
-      "core": true,
-      "unlocked_by": null,
-      "difficulty": 4,
-      "topics": [
+      slug: "pascals-triangle",
+      uuid: "d5caff8b-ffaa-4055-9bb9-7d31a4480956",
+      core: true,
+      unlocked-by: null,
+      difficulty: 4,
+      topics: [
+        "recursion",
         "list",
-        "math",
-        "recursion"
+        "math"
       ]
     },
     {
-      "slug": "atbash-cipher",
-      "uuid": "82169e85-3938-48da-ada9-7394adadc57a",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 2,
-      "topics": [
-        "cipher",
-        "string"
+      slug: "atbash-cipher",
+      uuid: "82169e85-3938-48da-ada9-7394adadc57a",
+      core: false,
+      unlocked-by: null,
+      difficulty: 2,
+      topics: [
+        "string",
+        "cipher"
       ]
     },
     {
-      "slug": "knapsack",
-      "uuid": "96f2a7e3-1283-4fca-944d-8cf2fe463c56",
-      "core": false,
-      "unlocked_by": null,
-      "difficulty": 6,
-      "topics": [
-        "dynamic_programming",
-        "greedy_programming",
-        "optimization"
+      slug: "knapsack",
+      uuid: "96f2a7e3-1283-4fca-944d-8cf2fe463c56",
+      core: false,
+      unlocked-by: null,
+      difficulty: 6,
+      topics: [
+        "optimization",
+        "dynamic-programming",
+        "greedy-programming"
       ]
     }
   ]
-}
+}
```

Comment:
I think this is a newline.  I didn't want to patch in the logic to handle removing the newline if the indentation level were 0.  I don't know if this is actually a big deal.

Reply:
Oh, it says no newline at end of file.

Reply:
So, regarding the differences here. configlet orders the topics in alphabetical order I think?

Anyway, we can easily get the right order by changing it in the `config.ss` file.

Reply:
Yeah, I think you're right it returns sorted order.

Reply:
In the `make-config` procedure it should be simple to sort it before it gets pretty printed.